test/support/tests_for_national_applicability.rb

```diff
@@ -256,6 +289,10 @@ def create_edition(attributes = {})
     create(edition_class.name.underscore, attributes)
   end
 
+  def build_edition(attributes = {})
+    build(edition_class.name.underscore, attributes)
+  end
```

Comment:
I don't see this being used, do we need it?

Reply:
thanks, use was removed but this slipped through.